CommonMC/src/SelectRecoMC_module.cc

```diff
@@ -229,12 +240,26 @@ namespace mu2e {
       tshmc._mom = mcstep.momentum();
       tshmc._time = fmod(mcstep.time(),_mbtime);
       tshmc._strawId = sdmc.strawId();
+      tshmc._earlyend = sdmc.earlyEnd();
+      // compute the signal propagation time and drift time
+      double vprop = 2.0*srep->halfPropV(sdmc.strawId(),1000.0*hit.energyDep());
```

Comment:
the 1000.0, which appears many places, would ideally be designed out, or maybe be a constexpr with a good name, or commented.

Reply:
This is an unfortunate unit convention conflict: Mu2e uses MeV, but tracker calibration uses KeV.  Fixing this out-of-scope for this PR.